src/environments/environment.ts

```diff
@@ -8,8 +8,8 @@
 import 'zone.js/dist/zone-error'; // Included with Angular CLI.
 import {Environment} from './environment.d';
 
-const apiHost = 'localhost:11111';
-const apiUrl = `http://${apiHost}/api/v1`;
+const apiHost = '127.0.0.1:3100'; // prism mock server address
+const apiUrl = `http://${apiHost}`;
```

Comment:
localhost in teoria è meglio di 127.0.0.1 e dovrebbe funzionare in maniera identica. 

io lascerei come apiHost "localhost:11111" (prism basta configurarlo usando l'opzione da terminale "prism mock --port 11111"
invece nell'apiUrl sono d'accordo su togliere il suffisso /api/ però v1 va tenuto. configurerò adeguatamente il server url nella specifica!

morale:
const apiHost = 'localhost:11111';
const apiUrl = `http://${apiHost}/v1`;

Reply:
Ma non credo di poter cambiare l'indirizzo

Reply:
dovrebbe essere lo stesso che compare nella chiave "servers:" della specifica openapi

Reply:
Cioè su stoplight non puoi specificare l'indirizzo a cui lo interroghi mi sa

Reply:
Almeno a me viene fuori in automatico quello che ho messo nell'environment

Reply:
Stessa cosa anche a Riccardo cestaro

Reply:
Adesso sto (finalmente) lavorando sull'api e provo a vedere se funziona. In caso lasciamo senza suffissi